fhir-smart/src/main/java/com/ibm/fhir/smart/AuthzPolicyEnforcementPersistenceInterceptor.java

```diff
@@ -51,20 +54,79 @@
 
     @Override
     public void beforeRead(FHIRPersistenceEvent event) throws FHIRPersistenceInterceptorException {
-        enforceDirectPatientAcess(event);
+        enforceDirectPatientAccess(event);
     }
 
     @Override
     public void beforeVread(FHIRPersistenceEvent event) throws FHIRPersistenceInterceptorException {
-        enforceDirectPatientAcess(event);
+        enforceDirectPatientAccess(event);
     }
 
     @Override
     public void beforeHistory(FHIRPersistenceEvent event) throws FHIRPersistenceInterceptorException {
-        enforceDirectPatientAcess(event);
+        enforceDirectPatientAccess(event);
     }
 
-    private void enforceDirectPatientAcess(FHIRPersistenceEvent event) throws FHIRPersistenceInterceptorException {
+    @Override
+    public void beforeSearch(FHIRPersistenceEvent event) throws FHIRPersistenceInterceptorException {
```

Comment:
thinking we should document what this does somewhere, but not sure where.  either here or at the class level i think.

Reply:
Documented at method level.